src/tests/multihost/ipa/test_adtrust.py

```diff
@@ -116,6 +116,45 @@ def test_pam_sss_gss_handle_large_krb_ticket(self, multihost,
         cmd = f'mv {bk_krbkcm} {krbkcm}'
         multihost.client[0].run_command(cmd, raiseonerr=False)
 
+    def test_adgrpwith_at_ratesign(self, multihost):
+        """
+        :title: user membership of AD group with @ sign
+        :id: ee9ca809-6ea7-48f2-a0fe-d9eccadf5d81
+        :customerscenario: True
+        :bugzilla:
+         https://bugzilla.redhat.com/show_bug.cgi?id=2061795
+        :description:
+         In the IPA-AD trust, IPA-user having group membership of an
+         AD-group, containing '@' sign in it's name, should be resolvable
+         with default re_expression on a ipa-client system.
+        :steps:
+          1. Create an AD-group having '@' sign in it's name
+          2. Create an AD-user and add it to above created group
+          3. From ipaclient, confirm ad-user is showing correct group
+             membership
+          4. From ipaclient, ad-group with '@' sign is correctly fetched
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+        """
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        ad_dmn = multihost.ad[0].domainname
+        ad = ADOperations(multihost.ad[0])
+        ad_user = 'aduser7'
+        ad_group = 'adgrp@7'
+        ad.create_ad_unix_user_group(ad_user, ad_group)
+        client.clear_sssd_cache()
+        getgrp = f'getent group {ad_group}@{ad_dmn}'
```

Comment:
In my opinion, if the command is run only once, it makes sense to inline the command directly to run_command without creating variables. 
It also prevents code warning R0914 Too many local variable.
Moreover if the command is inline it is less probable that after copy&paste it will end up like a crime against humanity  (yes I saw stuff like this):
```
sudo_cmd = "getent passwd %s@%s" % (user, domain)
```

Reply:
Added command directly

Reply:
Perfect.